### PR TITLE
fix: add packages list to lerna config for blackduck scanning

### PR DESCRIPTION
**Pull request by wattachai-lseg**
## Description

fix missing Blackduck incorrectly list dependencies of our packages dubbed BOM (bill of material) in its dashboard. 

Fixes https://jira.refinitiv.com/browse/ELF-2245

## Checklist

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [x] I have checked my code and corrected any misspellings

### CODE REVIEW

File: .github/workflows/blackduck_scan.yml [hunk 1]
```diff
@@ -42,10 +42,8 @@ jobs:
           --detect.code.location.name="${{ vars.BLACKDUCK_APP_ID }}-${{ vars.BLACKDUCK_PROJECT_ID }}-${VERSION_ID}" \
           --detect.source.path="." \
           --detect.clone.project.version.latest=true \
-          --detect.excluded.directories=appsec \
```
Comment (bualoy-napat) [inline .github/workflows/blackduck_scan.yml:45]: Why we need to remove this 3 lines ?

Comment (wattachai-lseg) [reply .github/workflows/blackduck_scan.yml:45]: these options have been removed. https://sig-product-docs.synopsys.com/bundle/integrations-detect/page/properties/all-properties.html

Comment (bualoy-napat) [reply .github/workflows/blackduck_scan.yml:45]: Ok!

**Review by bualoy-napat — DISMISSED** [review]
LGTM!

**Review by bualoy-napat — APPROVED** [review]

**Pull request merged**